CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to the Lacework Terraform Modules
+
+#### Table Of Contents
+
+[Before getting started?](#before-getting-started)
+
+[How to Contribute?](#how-can-i-contribute)
+* [Reporting Bugs](#reporting-bugs)
+* [Feature Requests](#feature-requests)
+* [Pull Requests](#pull-requests)
+
+[Developer Guidelines](/DEVELOPER_GUIDELINES.md)
+
+
+## Before getting started
+
+Read the [README.md](README.md) and
+ Hashicorps [best practices and syntax guidelines](https://www.terraform.io/docs/configuration/index.html)
+
+## Reporting Bugs
+
+Ensure the issue you are raising has not already been created under [issues](https://github.com/lacework/terraform-aws-cloudtrail/issues).
+If no current issue addresses the problem, open a new [issue](https://github.com/lacework/terraform-aws-cloudtrail/issues/new).
+Include as much relevant information as possible. See the [bug template](https://github.com/lacework/terraform-aws-cloudtrail/blob/main/.github/ISSUE_TEMPLATE/bug_report.md) for help on creating a new issue.
+
+## Feature Requests
+
+If you wish to submit a request to add new functionality or an improvement to a terraform module then use the the [feature request](https://github.com/lacework/terraform-aws-cloudtrail/blob/main/.github/ISSUE_TEMPLATE/feature_request.md) template to 
+open a new [issue](https://github.com/lacework/go-sdk/issues/new)
+
+## Creating a Pull Request
+
+If you have made a change or added new functionality, you can submit pull request. The project maintainers will aim to review in a reasonable timeframe. When submitting a pull request please read the [developer guidelines](/DEVELOPER_GUIDELINES.md)
```

Comment:
"e need to define what "reasonable timeframe" means, a week? would that be reasonable? 2 weeks?

Reply:
I think 2 weeks is reasonable

Reply:
I agree, and we can always adjust it but at least, it is a bit more clear.